fpdf/fpdf.py

```diff
@@ -35,6 +36,7 @@
 from fontTools import ttLib
 from fontTools import subset as ftsubset
 from io import BytesIO
+from fpdf.html import HTML2FPDF
```

Comment:
If you list `unescape` here, then the generic `import html` above can go away (assuming we don't have any other `unescape` that we need to distinguish it from).

Reply:
fixed!